python_modules/libraries/dagster-components/dagster_components/core/dsl_schema.py

```diff
@@ -24,8 +24,15 @@ class OpSpecBaseModel(BaseModel):
     tags: Optional[Dict[str, str]] = None
 
 
+def _post_process_key(rendered: Optional[str]) -> Optional[AssetKey]:
+    return AssetKey.from_user_string(rendered) if rendered else None
```

Comment:
yeah this is what I think the structure should be, but if we call this "resovling" rather than "rendering" this should be called a resolver